README.md

```diff
@@ -130,43 +130,48 @@ With your own parameters:
 
 <h2 align="center">Main Features</h2>
 
-**Splitting video by scenes for parallel encoding** because AV1 encoders are currently not very good at multithreading and encoding is limited to a very limited number of threads.
+Av1an allows for **splitting input video by scenes for parallel encoding** to improve encoding performance because most AV1 encoders are currently not very good at multithreading and encoding is limited to a very limited number of threads.
 
 - [Vapoursynth](http://www.vapoursynth.com) script input support.
 - Speed up video encoding.
-- Target Quality mode. Targeting end result reference visual quality. VMAF used as a substructure
+- "Target Quality" mode. Targeting end result reference visual quality. VMAF used as a substructure
 - Resuming encoding without loss of encoded progress.
 - Simple and clean console look.
 - Automatic detection of the number of workers the host can handle.
 - Both video and audio transcoding.
 
-## Install
+## Installation
 
+Av1an can be installed in two main ways, either:
 - With a package manager:
   - Cargo: `cargo install av1an`
   - Arch Linux: `pacman -S av1an`
+    
+- Or do it manually:
+  - Clone repository or download from [Releases](https://github.com/master-of-zen/Av1an/releases), then do `cargo build --release`.
 
-- Prerequisites:
+- Make sure to install these prerequisites first:
   - [Install FFmpeg](https://ffmpeg.org/download.html)
-  - [Vapoursynth](http://www.vapoursynth.com/)
-  - Recommended to install lsmash/ffms2 for faster and better processing
+  - [Install Vapoursynth](http://www.vapoursynth.com/)
+  - Recommended to install lsmash/ffms2 for faster and better pre-processing.
```

Comment:
I think using the term "pre-processing" in this context is not quite correct, since while vapoursynth *is* used for pre-processing, in this context we are talking about installing lsmash/ffms2 specifically to speed up piping frames to the encoder, which in my opinion doesn't really count as pre-processing since it doesn't actually modify the video data itself; it just decodes it.